DiscordBot/DiscordCrowdControl.py

```diff
@@ -0,0 +1,190 @@
+# Can't use real Crowd Control?  Get a taste of the experience with this!
+
+import asyncio
+import contextlib
+import datetime
+import json
+import random
+import socket
+import threading
+
+import discord
+from config import (AMMO, AUGS, CHANNEL_ID, COLOR, DESCRIPTION, EFFECTS,
+                    EFFECTS_COUNT, EFFECTS_HISTORY, FOOTER, GUILD_ID, HOST,
+                    INLINE, PORT, THUMBNAIL, TITLE, TOKEN, VOITING_TIME)
+
+intents = discord.Intents.default()
+bot = discord.Bot(intents=intents)
+reaction_emotes = [f"{i}\N{COMBINING ENCLOSING KEYCAP}" for i in range(1, EFFECTS_COUNT+1)]
+
+
+class CrowdControl:
+    def __init__(self, host, port) -> None:
+        self.socket = socket.create_server((host, port))
+        self.socket.settimeout(0.1)
```

Comment:
isn't a 100ms timeout too quick?

Reply:
Yes, it's for local sockets. I don't think it will make much difference